task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1196,7 +1196,7 @@ def watched_subprocess(self, mocker):
                 b"",
                 "task_instances.retry",
                 (),
-                {"id": TI_ID, "end_date": timezone.parse("2024-10-31T12:00:00Z")},
+                {"id": TI_ID, "end_date": timezone.parse("2024-10-31T12:00:00Z"), "rendered_map_index": None},
```

Comment:
I'd rather we didn't even send it if the task isn't mapped, and not send it when there isn't a template specified